SonarRunner.Shim/RoslynV1SarifFixer.cs

```diff
@@ -124,5 +129,38 @@ public static bool FixRoslynV1Sarif(string unfixedSarif, out string output)
                 }
             }            
         }
+
+        /// <summary>
+        /// Attempts to load and fix a SARIF file emitted by Roslyn 1.0 (VS 2015 RTM).
+       ///  Returns true if the file at the given path is valid a JSON file suitable for uploading to the server.
+        /// </summary>
+        public bool FixRoslynV1SarifFile(string sarifFilePath, ILogger logger)
```

Comment:
Having four public methods, three of which are marked as "for test" is a code smell. However, don't change it now. We can discuss this at some point later.